docs/Status_Reference.md

```diff
@@ -541,16 +541,36 @@ The following information is available in the `toolhead` object
   the printer had to be paused because the toolhead moved faster than
   moves could be read from the G-Code input.
 
+Additionally, if the printer uses `generic_cartesian` kinematic, the
+follwoing information is available:
+- `stepper_kinematics`: a map from a stepper name to the array of 3
+  floats - coefficients of the stepper kinematics for each of the axis
+  X, Y, and Z from the corresponding `kinematics` option, as indicated
+  by the specified carriages and their axes. Can be used to compute
+  stepper positions from the known toolhead position.
+- `toolhead_kinematics`: a map from a stepper name to the array of 3
+  floats - a contribution of a given stepper into the motion of each of
+  the axes X, Y and Z of the printer - which is effectively a
+  pseudoinverse of the `stepper_kinematics`. Can be used to compute
+  the toolhead position from the known stepper positions.
+
 ## dual_carriage
 
 The following information is available in
 [dual_carriage](Config_Reference.md#dual_carriage)
-on a cartesian, hybrid_corexy or hybrid_corexz robot
+on a cartesian, generic_cartesian, hybrid_corexy or hybrid_corexz robot
 - `carriage_0`: The mode of the carriage 0. Possible values are:
   "INACTIVE" and "PRIMARY".
 - `carriage_1`: The mode of the carriage 1. Possible values are:
   "INACTIVE", "PRIMARY", "COPY", and "MIRROR".
 
+If a user defined two dual carriages with `generic_cartesian` kinematic,
+the statuses exported are:
+- `carriage_0`: The array of length 2 - modes of the carriage 0 for X and Y
+  axes. Possible values are: "INACTIVE" and "PRIMARY".
+- `carriage_1`: The array of length 2 - modes of the carriage 1 for X and Y
+  axes. Possible values are: "INACTIVE", "PRIMARY", "COPY", and "MIRROR".
```

Comment:
What does "the array of length 2" mean?  Is there a reason this changes from the previous status format?

Reply:
The exported statuses have been updated per conversation here.